app/src/main/res/drawable/ic_chat_background.xml

```diff
@@ -0,0 +1,15 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Has this `VectorDrawable` been `avocado`ified?

Also, should it use color attributes, instead of hard-coded colors?